admin/src/WorkflowUI.js

```diff
@@ -564,21 +570,36 @@ class WorkflowUI extends Component {
 									.then( response => response.json() )
 									.then( data => ({ options: data }) )
 								}
-								value={recipient.value}
+								value={(recipient.users || []).filter(user => recipient.value.includes(String(user.id)))}
```

Comment:
This doesn't seem to account for `recipient.value` when it isn't an array, eg. if `recipient.multi` is false. Have you confirmed it works in both situations? It's a while since I worked on this, apologies if I'm missing something, but looks llike it can be the case in some situations. 

```suggestion
								value={( recipient.users || [] ).filter( user => recipient.value.includes( String( user.id ) ) )}
```

Reply:
@zeshanshani do you have time to resolve this particular issue? Would love to get this merged and released

Reply:
@roborourke I haven't tested that because I could only see that this specific field is always set to multi. Unless I'm wrong here?

Reply:
This is because the condition allows one or more users to select from so the field can never be just one user.